File: app/src/main/java/org/gears3d/gears3d/Gears3dSurfaceView.java
package org.gears3d.gears3d;

import android.content.Context;
import android.opengl.GLSurfaceView;

public class Gears3dSurfaceView extends GLSurfaceView {
    private final Gears3dRenderer mRenderer;
    public Gears3dSurfaceView(Context context) {
        super(context);

        // Create an OpenGL ES 2.0 context
        setEGLContextClientVersion(2);

        mRenderer = new Gears3dRenderer();

        // Set the Renderer for drawing on the GLSurfaceView
        setRenderer(mRenderer);
        setRenderMode(GLSurfaceView.RENDERMODE_WHEN_DIRTY);
    }
}

File: app/src/main/kotlin/org/gears3d/gears3d/Gears3dSurfaceView.kt
package org.gears3d.gears3d

import android.content.Context
import android.opengl.GLSurfaceView

class Gears3dSurfaceView(context: Context?) : GLSurfaceView(context) {
    private val mRenderer: Gears3dRenderer

    init {

        // Create an OpenGL ES 2.0 context
        setEGLContextClientVersion(2)
        mRenderer = Gears3dRenderer()

        // Set the Renderer for drawing on the GLSurfaceView
        setRenderer(mRenderer)
    }
}
